sphinx/source/docs/user_guide/server.rst

```diff
@@ -54,112 +57,104 @@ this.
     ></iframe>
     </div>
 
-When the controls are manipulated, their new values are automatically
-synced in the Bokeh server. Callbacks are triggered that also update the
-data for the plot in the server. These changes are automatically synced back
-to the browser, and the plot updates.
+Manipulating the UI controls communicates new values to the backend via Bokeh
+server triggering callbacks that update the visuals with the input in real
+time.
 
 .. _userguide_server_use_case:
 
-Use Case Scenarios
+Use case scenarios
 ------------------
 
-Now that we know what the Bokeh server is for, and what it is capable of
-doing, it's worth considering a few different scenarios when you might
-want to use a Bokeh Server.
+Consider a few different scenarios when you might want to use the Bokeh server.
 
 .. _userguide_server_use_case_individual:
 
-Local or Individual Use
+Local or individual use
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-One way that you might want to use the Bokeh server is during exploratory
-data analysis, possibly in a Jupyter notebook. Alternatively, you might
-want to create a small app that you can run locally, or that you can send
-to colleagues to run locally. The Bokeh server is very useful and easy to
-use in this scenario. Both of the methods here below can be used effectively:
+You might want to use the Bokeh server for exploratory data analysis, possibly
+in a Jupyter notebook, or for a small app that you and your colleagues can run
+locally. The Bokeh server would be very convenient here, allowing for effective
+use of the following methods:
 
 * :ref:`userguide_server_bokeh_client`
```

Comment:
I'm not sure the server_bokeh_client ref is best here, today `bokeh.client` is for a very specific use-case of customizing individual app sessions (e.g. on a per-user basis) People used to use `bokeh.client` to try and have apps in "outside" python processes but that practice is actively anti-encouraged, and in fact some of the user-facing APIs people used for that are/have been removed. tldr we should not point people anywhere near `bokeh.client` for 'getting started"